s1reader/s1_etad.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>I think it would be more informative if we mention that the left column is range components, and the right column is azimuth components of corrections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/opera-adt/notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>It is in the bottom X-axis label. I make it a little bigger, hoping it's more obvious.</p>